openpype/modules/kitsu/plugins/publish/integrate_kitsu_note.py

```diff
@@ -142,8 +134,19 @@ def process(self, context):
         self.get_settings_from_context(context)
 
         for instance in context:
+            self.log.info(instance.data)
+            self.log.info([instance for instance in context])
+            if any(
+                instance.data.get("kitsu_comment")
+                for instance in context
+            ):
+                self.log.info(
+                    "Kitsu comment already set, skip comment creation."
+                )
+                break
```

Comment:
If we do as you say, we'll always have at least two reviews done by the IntegrateKitsuNote.
One for the review and one for the workfile.
Otherwise, we need to add back a Setting to avoid doing it for the workfile family...
But that caused problems in the past.

Reply:
Updated